src/eduid_common/api/utils.py

```diff
@@ -67,7 +71,24 @@ def retrieve_modified_ts(user, dashboard_userdb):
         user, dashboard_user, dashboard_user.modified_ts))
 
 
-def save_dashboard_user(user):
+def get_dashboard_user():
```

Comment:
This will not return a dashboard user what I can see. It will return a user document from the central db loaded in a User class.